tf2/include/tf2/exceptions.h

```diff
@@ -40,17 +40,34 @@
 namespace tf2
 {
 
+#if defined(_WIN32)
+#pragma push_macro("NO_ERROR")
+#undef NO_ERROR
+#endif
+
 enum class TF2Error : std::uint8_t
 {
-  NO_ERROR = 0,
-  LOOKUP_ERROR = 1,
-  CONNECTIVITY_ERROR = 2,
-  EXTRAPOLATION_ERROR = 3,
-  INVALID_ARGUMENT_ERROR = 4,
-  TIMEOUT_ERROR = 5,
-  TRANSFORM_ERROR = 6
+  TF2_NO_ERROR = 0,
```

Comment:
You might want a comment here explaining why the `TF2_` prefix is here, lest we have someone propose to switch them back unwittingly.

Reply:
Good point.  Comment added in d711136